MediaBrowser.Api/Session/SessionsService.cs

```diff
@@ -230,6 +230,17 @@ public class PostFullCapabilities : ClientCapabilities, IReturnVoid
         public string Id { get; set; }
     }
 
+    [Route("/Sessions/Viewing", "POST", Summary = "Reports that a session is viewing an item")]
```

Comment:
who's using this endpoint (or going to use)?

Reply:
I figure this is for EmbyStat?

Reply:
I was planning on adding this to the web client as well.